src/main.py

```diff
@@ -452,7 +453,7 @@ def process_response(response):
 			for directive in j['messageBody']['directives']:
 				if directive['namespace'] == 'SpeechSynthesizer':
 					if directive['name'] == 'speak':
-						player.play_speech(mrl_fix("file://" + tmp_path + directive['payload']['audioContent'].lstrip("cid:") + ".mp3"))
+						player.play_speech(mrl_fix("file://" + tmp_path + hashlib.md5(directive['payload']['audioContent'].lstrip("cid:")).hexdigest() + ".mp3"))
```

Comment:
replace lstrip("cid:") with replace("cid:","",1) to avoid the case where lstrip strips cid:c or cid:i or cid:d